client/jetpack-cloud/sections/partner-portal/sidebar/index.tsx

```diff
@@ -55,7 +54,7 @@ class PartnerPortalSidebar extends Component< Props > {
 							} ) }
 							link="/partner-portal/licenses"
 							onNavigate={ this.onNavigate( 'Jetpack Cloud / Partner Portal / Licenses' ) }
-							selected={ itemLinkMatches( '/partner-portal/licenses', path ) }
+							selected={ path === '/partner-portal/licenses' }
```

Comment:
When we use that strict comparison, some URLs will end up not highlighted, e.g.:

- http://jetpack.cloud.localhost:3000/partner-portal/licenses/unassigned
- http://jetpack.cloud.localhost:3000/partner-portal/licenses/assigned
- http://jetpack.cloud.localhost:3000/partner-portal/licenses/revoked

The same is valid for Payment Methods as well:
- http://jetpack.cloud.localhost:3000/partner-portal/payment-methods/add

We can switch to some pattern matching that would highlight `/licenses/*` routes as well.